### PR TITLE
Formatting and Interfaces

### PR DESCRIPTION
**Pull request by themandalore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,11 +31,14 @@
     "hardhat-deploy-ethers": "^0.3.0-beta.7",
     "hardhat-gas-reporter": "^1.0.3",
     "hardhat-log-remover": "^2.0.0",
+    "prettier": "^2.2.1",
+    "prettier-plugin-solidity": "^1.0.0-beta.7",
```
Comment (krasi-georgiev) [inline package.json:35]: You would want these globally installed not locally here.

File: package.json [hunk 2]
```diff
@@ -31,11 +31,14 @@
...
     "tellorplayground": "github:tellor-io/tellorplayground#main"
   },
   "dependencies": {
     "dotenv": "^8.2.0",
     "ethers": "^5.0.23",
-    "openzeppelin-solidity": "github:OpenZeppelin/openzeppelin-contracts#master"
+    "openzeppelin-solidity": "github:OpenZeppelin/openzeppelin-contracts#master",
+    "typescript": "^4.2.3"
```
Comment (krasi-georgiev) [inline package.json:42]: Was this added by mistake or it is needed for something?

Comment (themandalore) [reply package.json:42]: hmmm...I don't remember adding it...yeah take it out

File: package.json (hunk 1 above)
Comment (themandalore) [reply package.json:35]: why globally: https://github.com/prettier-solidity/prettier-plugin-solidity

doesn't it say to do locally here?

Comment (krasi-georgiev) [reply package.json:35]: installing it globally allows to use it with vscode on any project, not needing to install for every single project.

I don't even remember installing, when I installed the Solidity vscode extension is got installed globally as part of that.
https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity

jump to : Formatting using Prettier and the Prettier Solidity Plugin

**Pull request merged**